src/main/java/org/tools4j/hoverraft/message/direct/DirectCommandMessage.java

```diff
@@ -96,4 +96,11 @@ public void unwrap() {
         command.unwrap();
         super.unwrap();
     }
+
+
+    @Override
+    public void copyFrom(final CommandMessage commandMessage) {
```

Comment:
it implements (@Overrides) void copyFrom(final CommandMessage commandMessage) defined in the CommandMessage interface by calling 
protected method copyFrom(final DirectPayload directPayload).

I did it for the reason that I don't want to expose copyFrom(final DirectPayload directPayload) as it will allow to copy bytes of different types. But I want to allow copyFrom() on the same types.

So, I defined (an "overloaded") void copyFrom(CommandMessage commandMessage) in the CommandMessage interface and implemented that by calling protected method copyFrom(final DirectPayload directPayload).